README.md

```diff
@@ -1,50 +1,67 @@
 # SignalFx .NET Lambda Wrapper
 
-## Overview
+The SignalFx .NET Lambda Wrapper wraps around an AWS Lambda .NET or ASP.NET Core
+function handler, which enables you to send metrics and traces to Splunk APM.
 
-You can use this document to add a SignalFx wrapper to your AWS Lambda for .NET.
+The Lambda wrapper manually instruments the Lambda function itself, and not
+any libraries or frameworks. To get traces from libraries and frameworks in
+the Lambda function, manually instrument them.
 
-The SignalFx C# Lambda Wrapper wraps around an AWS Lambda .NET function handler, which allows metrics and traces to be sent to SignalFx.
+See these links for information about and examples of manual instrumentation:
 
-At a high-level, to add a SignalFx .NET Lambda wrapper, you can:
-   * Package the code yourself; or
-   * Use a Lambda layer containing the wrapper, and then attach the layer to a Lambda function.
+- https://github.com/signalfx/tracing-examples/tree/master/dotnet-manual-instrumentation
+- https://github.com/signalfx/tracing-examples/tree/master/signalfx-tracing/signalfx-dotnet-tracing
 
-To learn more about Lambda Layers, please visit the [AWS documentation site](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
+There are two options to use the SignalFx Lambda wrapper:
 
-## Step 1: Add the wrapper to the project
+- Manually deploy the Lambda wrapper to your code
+- Use a Lambda layer that Splunk hosts
 
-Add the [`signalfx-lambda-functions` NuGet package](https://www.nuget.org/packages/signalfx-lambda-functions/) to your project.
+## Manually deploy the Lambda wrapper to your code
 
-For advanced users who want to reduce the size of deployment packages, you can use the package as a developer dependency, but in production, you would add the wrapper to a layer in the Lambda environment. This option allows you to work with the wrapper in a local setting and reduce the size of deployment packages at the same time.
+Follow these steps to add the SignalFx Lambda wrapper for .NET or ASP.NET Core. 
 
-### Option 1: For AWS Serverless ASP.NET Core Lambda
+### Step 1. Add the wrapper to your project
 
-In this option, perform the following steps:
+Add the [`signalfx-lambda-functions` NuGet package](https://www.nuget.org/packages/signalfx-lambda-functions/)
+to your project. If you want to reduce the size of your deployment, you can
+add the package as a developer dependency, but in a production environment you
+should add the wrapper as a layer to reduce the deployment size.
```

Comment:
I would rather remove the comment about the layer here because the steps are independent: no matter if one uses the layer or not: adding the package is the same in any case.

```suggestion
to your project.```